BibBuddy/app/src/main/java/de/bibbuddy/AuthorFragment.java

```diff
@@ -128,43 +131,77 @@ private void deleteAuthors() {
     if (selectedAuthorItems.size() > 1) {
       alertDeleteAuthor.setTitle(R.string.delete_authors);
       alertDeleteAuthor.setMessage(
-          getString(R.string.delete_authors_message) + " " + getString(R.string.delete_warning));
+          getString(R.string.delete_authors_message)
+              + convertAuthorListToString(selectedAuthorItems)
+              + getString(R.string.finally_delete) + " "
+              + getString(R.string.delete_warning));
     } else {
       alertDeleteAuthor.setTitle(R.string.delete_author);
       alertDeleteAuthor.setMessage(
-          getString(R.string.delete_author_message) + " " + getString(R.string.delete_warning));
+          getString(R.string.delete_author_message)
+              + convertAuthorListToString(selectedAuthorItems)
+              + getString(R.string.finally_delete) + " "
+              + getString(R.string.delete_warning));
```

Comment:
Maybe create a separate function for this.

Reply:
Can not be sensibly decomposed here.